httpx/_client.py

```diff
@@ -1800,7 +1811,7 @@ def __init__(
         self.timeout = timeout
         self.close_client = close_client
 
-    def __enter__(self) -> "Response":
```

Comment:
I don't think there's any reason to change this line.

Reply:
Yeah, if nobody is going to subclass StreamContextManager

Reply:
Will revert this part back